backend/testUtils/school.ts

```diff
@@ -41,6 +41,42 @@ export const testSchools = [
   },
 ];
 
+// set up test schools
+export const testSchools2 = [
```

Comment:
why did we make a new test schools variable?

Reply:
I wanted to have a new set of test schools with different countries to make sure the filtering works. Should I combine them into the existing test school array? I'm scared doing so might break the older test cases

Reply:
you could update the original `testSchools` to have `some-country1` and `some-country2` and that shouldn't break anything. you could also add a third entry with `some-country1` again. just update the existing tests if something fails

Reply:
nvm we can keep separate variables for more clarity? i think that's what we're going for based on other PRs

Reply:
okay, sounds good!